hack/verify-linkcheck.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# For running in Jenkins, get golang into our PATH so we can run e2e.go
+export PATH=${PATH}:/usr/local/go/bin
```

Comment:
also I think I may have misled you. this should probably go in the job config, not this script, since this script is not jenkins-specific.